benches/lib.rs

```diff
@@ -59,3 +59,12 @@ fn is_subset_4(b: &mut Bencher) {
     let sup: RoaringBitmap = (0..65536).collect();
     b.iter(|| test::black_box(sub.is_subset(&sup)))
 }
+
+#[bench]
+fn remove_range_bitmap(b: &mut Bencher) {
+    let mut sub: RoaringBitmap = (0..65536).collect();
+    b.iter(|| {
+        test::black_box(sub.remove_range(4096 + 1, 65536));
```

Comment:
This closure is going to be called many times for each `sub` that is created, so only the first call will actually remove values from it. I think the benchmark will have to clone the bitmap every time, hopefully the time for cloning is enough less than the time for removing a range to make the benchmark still useful.